chartmogul.go

```diff
@@ -157,6 +167,13 @@ type AnchorCursor struct {
 	StartAfter string `json:"start-after,omitempty"`
 }
 
+// PaginationWithCursor is the new standard for cursor with pagination.
+type PaginationWithCursor struct {
+	PerPage uint32 `json:"per-page,omitempty"`
+	// Cursor is used to get the next set of Entries and its value should be the UUID of last Entry from previous response.
+	Cursor string `json:"cursor,omitempty"`
+}
```

Comment:
I'm creating a new type because `AnchorCursor` already exists which is pretty much similar in implementation. I guess if we ever have a plan to migrate all paginations, we should standardize it to one type and I guess this should be the format (based on the internal discussion in Notion)

Reply:
Well, adding a 4th cursor struct is not great, but it's backwards compatible.

We'll have to support the old pagination for a while as well. Since these new endpoints don't have legacy pagination, we don't have to implement it yet, but when we need it, we'll have to do something like an `interface` that accepts both old and new cursor structs.